### DISCUSSION TITLE
Set Book Posters to Series Poster

### DISCUSSION THREAD

**Discussion by yacob841** [gotson/komga#1747] [Ideas]
I saw a similar request where someone was asking a question for how to bulk change the book posters to the series poster and the response was to script it. This might get the same response but I’m more so asking for it to be a feature request. Similar to how you have the feature for setting the series poster to be what chapter, it would be nice to have a setting where the book poster can be the first image of each book or the series poster.

**Comment by sgayda2** [comment]
I would also appreciate having the book poster show up as the series poster for mangas, where the first page of the 'book' is really just the first panel and not a special cover. In these cases it would make more sense to instead show the series poster for all of the books as they are really chapters of the manga and don't need/have their own unique covers most of the time.

This can be a setting so as not to force this on users or series that don't need it but it would be helpful to have.
